openlibrary/plugins/worksearch/subjects.py

```diff
@@ -162,6 +162,16 @@ def normalize_key(self, key):
     def process_key(self, key):
         return key
 
+class subject_history(delegate.page):
+    path = '/subjects/history'
+    encoding = 'json'
+    
+    @jsonapi
+    def GET(self,key):
+        i = web.input()
+        if i.get("limit")>100:
+            return "I'm sorry Dave, I'm afraid I can't do that."
```

Comment:
Is returning a string possible here? Just curious.

Reply:
Nope :+1: , the `@jsonapi` decorator is expecting a dictionary / json serializable type

Reply:
it now returns an error dict